src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -456,7 +461,8 @@ private static bool NeedsSeparator(SyntaxToken token, SyntaxToken next)
 
             if (token.IsKind(SyntaxKind.ColonToken))
             {
-                return !token.Parent.IsKind(SyntaxKind.InterpolationFormatClause);
+                return !token.Parent.IsKind(SyntaxKind.InterpolationFormatClause)
+                    && !token.Parent.IsKind(SyntaxKind.XmlPrefix);
```

Comment:
Nit: formatting feels weird here. I'd expect the `&&` to either be in line with the previous condition, or on the previous line.

Reply:
@333fred Agree. I was only being consistent with the above formatting:

https://github.com/dotnet/roslyn/blob/021549aa433a48c0476bf88af54577888e31333a/src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs#L455-L457

I'll stick to place `&&` at the end of previous line instead of start of new line.